--- a/files/usr/share/cinnamon/applets/calendar@cinnamon.org/applet.js	2012-03-12 14:31:34.000000000 +0000
+++ b/files/usr/share/cinnamon/applets/calendar@cinnamon.org/applet.js	2012-03-13 10:56:47.918048379 +0000
@@ -62,9 +62,7 @@ MyApplet.prototype = {
             this._calendar = new Calendar.Calendar(this._eventSource);       
             vbox.add(this._calendar.actor);
 
-            let item = new PopupMenu.PopupMenuItem(_("Date and Time Settings"))
-            item.connect("activate", Lang.bind(this, this._onLaunchSettings));
-            //this.menu.addMenuItem(item);
+            let item = this.menu.addSettingsAction(_("Date and Time Settings"), 'gnome-datetime-panel.desktop');
             if (item) {
                 let separator = new PopupMenu.PopupSeparatorMenuItem();
                 separator.setColumnWidths(1);
@@ -105,11 +103,6 @@ MyApplet.prototype = {
         this.menu.toggle();
     },
     
-    _onLaunchSettings: function() {
-        this.menu.close();
-        Util.spawnCommandLine("cinnamon-settings calendar");
-    },
-
     _updateClockAndDate: function() {
         let displayDate = new Date();
         let dateFormattedFull = displayDate.toLocaleFormat(this._dateFormatFull);
