Index: linuxmint-Cinnamon-5ab432d/files/usr/lib/cinnamon-settings/cinnamon-settings.py
===================================================================
--- linuxmint-Cinnamon-5ab432d.orig/files/usr/lib/cinnamon-settings/cinnamon-settings.py
+++ linuxmint-Cinnamon-5ab432d/files/usr/lib/cinnamon-settings/cinnamon-settings.py
@@ -20,7 +20,7 @@ try:
     import lxml.etree
     import locale    
     import imtools
-    import Image
+    from PIL import Image
     import tempfile
     import math
     import cgi
Index: linuxmint-Cinnamon-5ab432d/files/usr/lib/cinnamon-settings/imtools.py
===================================================================
--- linuxmint-Cinnamon-5ab432d.orig/files/usr/lib/cinnamon-settings/imtools.py
+++ linuxmint-Cinnamon-5ab432d/files/usr/lib/cinnamon-settings/imtools.py
@@ -18,10 +18,10 @@ from cStringIO import StringIO
 from itertools import cycle
 from urllib import urlopen
 
-import Image
-import ImageDraw
-import ImageEnhance
-import ImageOps, ImageChops, ImageFilter
+from PIL import Image
+from PIL import ImageDraw
+from PIL import ImageEnhance
+from PIL import ImageOps, ImageChops, ImageFilter
 
 ALL_PALETTE_INDICES = set(range(256))
 CHECKBOARD = {}
